src/lib/components/wizard-navigation-bar.component.horizontal.less

```diff
@@ -271,39 +250,35 @@ aw-wizard-navigation-bar.horizontal {
       pointer-events: none;
       text-align: center;
 
-      a {
-        cursor: pointer;
-
-        .label {
-          display: inline-block;
-          padding-top: @text-padding-bottom;
-          color: @wz-color-current;
-          line-height: @text-height;
-          font-size: @text-height;
-          text-decoration: none;
-          text-transform: uppercase;
-          text-align: center;
-          font-weight: bold;
-          transition: 0.25s;
-        }
+      a .label {
+        display: inline-block;
+        padding-top: @text-padding-bottom;
+        color: @wz-color-current;
+        line-height: @text-height;
+        font-size: @text-height;
+        text-decoration: none;
+        text-transform: uppercase;
+        text-align: center;
+        font-weight: bold;
+        transition: 0.25s;
       }
     }
 
-    li a:hover .label {
-      color: darken(@wz-color-current, 20%);
-    }
+    li.navigable {
 
-    // default steps shouldn't change when hovered, because they aren't clickable
-    li.default a {
-      cursor: auto;
+      // TODO: The possibility to navigate to a step via the navigation bar should be governed by the code, not by CSS.
+      // Removing this definition now will allow to navigate to every step the user is able to navigate with the [awGoToStep] directive.
+      // For example, right after opening the wizard with basic configuration, the user would be able to go to the second step
+      // by clicking its indicator in the navigation bar instead of using the [Next] button.  This behavior is undesireable.
+      pointer-events: auto;
```

Comment:
Mind the comment. I wanted to get rid of the `pointer-events` styling on my way, but could not.

Reply:
Hmmm, I'm not sure I understand the problem.

I mean the current navigation mode interface contains two methods to check whether a navigation from a current step to a destination step can be performed:
- `canGoToStep`: this method is called for "normal" transitions, which are normally performed when the navigation directives are used
- `isNavigable`: this method performs an additional check for the navigation using the navigation bar

To disable the navigation using the navigation bar we should just need to improve the `isNavigable` method, right?

Reply:
> To disable the navigation using the navigation bar we should just need to improve the `isNavigable` method, right?

The method itself is fine, the problem is that it is only called to set CSS classes. It is not used when a user clicks a step indicator, and I believe it should be.

When checking whether the user can navigate to a step, we should not rely on CSS classes that are supposed to set `pointer-events: none` to silence the [awGoToStep] directive (the one used by the navigation bar internally). Instead, we should probably make the [awGoToStep] conditional so that it only takes effect when needed, no matter what is happening on the presentation level (CSS).

Perhaps, users will also find use cases for conditional goTo*Step directives.

Having said that, I think the best action now is to remove this comment and leave `pointer-events: none` intact because this change:
1. has little to do with the purpose of this PR and is better to be done separately / later
2. has no purpose from the perspective of the end users.
3. will affect the `NavigationMode`'s part of the code source code, which is already being refactored in https://github.com/madoar/angular-archwizard/pull/166 and https://github.com/madoar/angular-archwizard/pull/178, and will complicate matters even further.

Shall I remove the comment?

Reply:
I agree.
Let's remove the comment and solve the problem in another PR/issue.

Reply:
Done in b9b48b5d